packages/rs-platform-version/src/version/dpp_versions/dpp_state_transition_versions/v3.rs

```diff
@@ -0,0 +1,46 @@
+use crate::version::dpp_versions::dpp_state_transition_versions::{
+    AddressFundsTransitionVersions, ContractTransitionVersions, DPPStateTransitionVersions,
+    DocumentTransitionVersions, DocumentsBatchTransitionValidationVersions,
+    DocumentsBatchTransitionVersions, IdentityCreditWithdrawalTransitionVersions,
+    IdentityTransitionAssetLockVersions, IdentityTransitionVersions,
+};
+
+pub const STATE_TRANSITION_VERSIONS_V3: DPPStateTransitionVersions = DPPStateTransitionVersions {
```

Comment:
Please comment with changelog, eg.

```suggestion
/// State transition version 3.
///
/// ## Changes since v2:
///
/// * `calculate_min_required_fee_on_identity_create_transition` changed from 0 to 1
/// * `calculate_min_required_fee_on_identity_top_up_transition` changed from 0 to 1
pub const STATE_TRANSITION_VERSIONS_V3: DPPStateTransitionVersions = DPPStateTransitionVersions {
```